drv/transceivers-server/src/udp.rs

```diff
@@ -600,9 +649,20 @@ impl ServerImpl {
         // any modules at index 32->63 are not currently supported.
         let invalid_modules = ModuleId(0xffffffff00000000);
```

Comment:
There's an `all_sidecar()` helper function to return the lower 32 bits, so you could use that and negate it.

Reply:
I don't see a function with this name anywhere in Hubris, can you point me to it?